### PR TITLE
Add lint flake8

### PR DESCRIPTION
**Pull request by wakiyamap**
I changed it pass through lint of flake8.
I also added flake8 to github actions.

P.S. Please check the DM of twitter.

### CODE REVIEW

File: counterpartylib/lib/messages/triggers/asset_metadata.py [hunk 1]
```diff
@@ -115,9 +115,11 @@ def __query(self, tx, asset, query_type, bson_object):
             problems += self.__lock(tx, asset, bson_object)
         elif query_type == TYPE_DELETE:
             problems += self.__store(tx, asset, None)
-        elif dry_run:
-            problems += 'Unknown query type'
+        #TODO
+        #elif dry_run:
+        #    problems += 'Unknown query type'
```
Comment (cryptcoin-junkey) [inline counterpartylib/lib/messages/triggers/asset_metadata.py]:
```suggestion
        elif tx is None:
            problems += 'Unknown query type'
```

File: counterpartylib/lib/messages/triggers/asset_metadata.py [hunk 2]
```diff
@@ -115,9 +115,11 @@ def __query(self, tx, asset, query_type, bson_object):
...
         else:
+            problems += 'Unknown query type'
```
Comment (cryptcoin-junkey) [inline counterpartylib/lib/messages/triggers/asset_metadata.py:119]:
```suggestion
```

**Review by cryptcoin-junkey — CHANGES_REQUESTED** [review]

**Comment by cryptcoin-junkey** [comment]
@wakiyamap Thank you for your contribution. Please have a look to my change requests.

**Comment by wakiyamap** [comment]
I fixed it.

**Pull request merged**